README.md

```diff
@@ -46,33 +48,16 @@ cd stylised-controllable-image-captioning-StreamlitApp
 
 Build and run the docker image locally:
 ```
-docker build -t ctrl-img-cap-streamlit:latest .
-docker run -d -p 8501:8501 ctrl-img-cap-streamlit:latest
+make run
 ```
 Navigate to http://localhost:8501 for the app. (Streamlit runs on port 8501 by default)
 
-*Note : When you'll run the container for the first time it will download a resnet101 model. The Streamlit app is run in CPU.*
-
-## Install and Run in Conda (Alternative)
```

Comment:
I am thinking an installation guide on Anaconda is useful for users without Docker.
For the docker build, are we relying on requirements.txt to install packages?
If so, maintaining Anaconda launch options won't be too troublesome coz it only relies on requirements.txt

Reply:
> For the docker build, are we relying on requirements.txt to install packages?

We are. The troublesome bit is dealing with model artefacts: I want the manual download description to be gone, so that people new to the project don't have to think and won't get confused about it.

The Conda description conflicts with that goal.

Additionally, singling out Conda seems arbitrary. People might want to use Pipenv or Poetry or virtualenv or ... instead. And if they have already made this decision, they know enough about the setup to do this without instruction.